src/endpoints/sources.js

```diff
@@ -391,6 +403,29 @@ async function sendAny(req, res, url, accept) {
 }
 
 
+/**
+ * Generate an Express endpoint that receives a group overview
+ * determined by the request.
+ *
+ * @param {sourceExtractor} sourceExtractor - Function to provide the Source instance from the request
```

Comment:
Ditto about not needing `sourceExtractor` here either.

Reply:
Removed: 84e61b76ce5b59ee5e9fab51f42bfc19733bf4c6

Reply:
also 899a5e9cc73358a48969fefa0c38eaa0174b79d6